spec/providers/analytics.spec.ts

```diff
@@ -69,22 +69,33 @@ describe('Analytics Functions', () => {
       it('should handle an event with the appropriate fields', () => {
         const cloudFunction = analytics
           .event('first_open')
-          .onLog((data: analytics.AnalyticsEvent) => data);
+          .onLog(
+            (data: analytics.AnalyticsEvent, context: EventContext) => data
+          );
 
         // The event data delivered over the wire will be the JSON for an AnalyticsEvent:
         // https://firebase.google.com/docs/auth/admin/manage-users#retrieve_user_data
-        let event: LegacyEvent = {
-          eventId: 'f2e2f0bf-2e47-4d92-b009-e7a375ecbd3e',
-          eventType: 'providers/google.firebase.analytics/eventTypes/event.log',
-          resource: 'projects/myUnitTestProject/events/first_open',
+        let event: Event = {
```

Comment:
don't need to handle it in this pr but can these lets be consts? In other tests as well.

Reply:
Yeah, good idea. I'll do it in a separate PR.